cpp/src/phonenumbers/utf/unicodetext.cc

```diff
@@ -505,11 +505,11 @@ UnicodeText::const_iterator UnicodeText::MakeIterator(const char* p) const {
   return const_iterator(p);
 }
 
-string UnicodeText::const_iterator::DebugString() const {
+std::string UnicodeText::const_iterator::DebugString() const {
```

Comment:
Can we just use a using statement? It is the .cc file.

Reply:
Yah true that even better option. Done and tested.